playbooks/roles/mail/templates/mailman3-nginx-proxy.conf.j2

```diff
@@ -1,7 +1,7 @@
 # {{ ansible_managed }}
 
 upstream mailman3 {
-    server unix:/run/mailman3/web/uwsgi.sock fail_timeout=0;
+    server unix:/run/mailman3-web/uwsgi.sock fail_timeout=0;
```

Comment:
The location of this socket has changed in either ubunutu's config or the upsteream itself.